test/Refinancing.ts

```diff
@@ -829,7 +829,7 @@ describe("Refinancing", () => {
             const newLenderOwes = refiLoanTerms.principal.add(interestDue);
 
             await mint(mockERC20, newLender, newLenderOwes);
-            await approve(mockERC20, newLender, loanCore.address, newLenderOwes);
+            await approve(mockERC20, newLender, refinanceController.address, newLenderOwes);
```

Comment:
Were these tests not passing before? Just curious why the approvals needed to change based on the addition of a revert statement as the only modification

Reply:
In the tests where refinancing is successful, the approval is made to the refi contract, which is the correct way to do this. So this change is just showing that same flow so it is not a variant in all these tests where there is an expected revert that is not related to a missing approval.

Reply:
Ahh I see, so it should have always been this way?

Reply:
Yes, I would argue that it should have been this way. It doesnt affect the test outcomes, but more shows that the revert msg is not due to the wrong approval.